### PR TITLE
complete header and css box #3

### PR DESCRIPTION
**Pull request by DatWithTheWorld**
Desktop: 
image

### CODE REVIEW

File: src/layouts/header.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<<<<<<< Updated upstream
```
Comment (xuanquocc) [inline src/layouts/header.html:1]: clear this

File: src/stylesheets/components/_index.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<<<<<<< Updated upstream
+=======
+@forward "box";
+@forward "navList";
+@forward "button";
+@forward "buttonnocl";
+>>>>>>> Stashed changes
```
Comment (xuanquocc) [inline src/stylesheets/components/_index.scss:7]: clear this

**Review by xuanquocc — CHANGES_REQUESTED** [review]
end of lines

File: src/components/box.html [hunk 3]
```diff
@@ -20,10 +20,10 @@ <h3 class="content__secondary__line-h3">{{ item.secondaryLines }}</h3>
             </include>
             </div>
             </div>
-            <picture class="image">
-                <source class="image__img" media="(min-width:1200px)" srcset="../assets/img/{{ item.imageLink1 }}.png" alt="{{ item.imageDesc1 }}"/>
-                <source class="image__img" media="(min-width:429px)" srcset="../assets/img/{{ item.imageLink2 }}.png" alt="{{ item.imageDesc2 }}"/>
-                <img class="image__img" src="../assets/img/{{ item.imageLink3 }}.png" alt="{{ item.imageDesc3 }}" />
-            </picture>
+            <figure class="image">
```
Comment (baonguyen26) [inline src/components/box.html:23]: Why did you change the code here?

**Pull request merged**